dev-util/rgbds/rgbds-0.6.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="Rednex Game Boy Development System"
+HOMEPAGE="https://rgbds.gbdev.io/"
+if [[ "${PV}" == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/gbdev/${PN}.git"
+else
+	SRC_URI="https://github.com/gbdev/${PN}/archive/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="ZLIB"
+SLOT="0"
+
+DEPEND="media-libs/libpng"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_compile() {
+	append-cflags -DNDEBUG
```

Comment:
This is probably append-cppflags instead?

Reply:
Good point, but the project doesn't use `CPPFLAGS` so it has to be `CFLAGS`.

Reply:
Actually it needs to be `append-flags` because it needs to be set for both `CFLAGS` and `CXXFLAGS`. I changed it.